onnxruntime/core/graph/model.cc

```diff
@@ -394,14 +394,14 @@ Status Model::LoadFromBytes(int count, void* p_bytes, /*out*/ ONNX_NAMESPACE::Mo
 
 Status Model::LoadFromBytes(int count, void* p_bytes, /*out*/ std::shared_ptr<Model>& p_model,
                             const IOnnxRuntimeOpSchemaRegistryList* local_registries, const logging::Logger& logger) {
-  ModelProto model_proto;
+  auto model_proto = onnxruntime::make_unique<ModelProto>();
```

Comment:
Does the model_proto member inside the Model class have to be heap allocated? It's forcing all call sites to heap allocate as well just like this particular line. ModelProto has a move constructor that can be used to move the local object to the Model class. Doesn't have to be fixed in this PR.

Reply:
Thanks. I ll address that in a separate PR.